app/sprinkles/core/src/Util/CheckEnvironment.php

```diff
@@ -257,6 +263,39 @@ public function checkPdo()
         return $problemsFound;
     }
 
+    /**
+    * Check that log, cache, and session directories exist.
+    */
+    public function checkDirectories()
+    {
+        $problemsFound = false;
+
+        $directoryPaths = [
+            "logs"     => $this->locator->findResource('log://'),
+            "cache"    => $this->locator->findResource('cache://'),
+            "sessions" => $this->locator->findResource('session://')
+        ];
+
+        foreach ($directoryPaths as $directory => $path) {
+          if ($path == null) {
+            $problemsFound = true;
+            $this->resultsFailed['directory-' . $directory] = [
+                'title'   => "<i class='fa fa-file-o fa-fw'></i> A required directory was not found.",
+                'message' => "Please check that <code>userfrosting/app/$directory</code> exists.",
+                'success' => false
+            ];
+          }
+          else {
```

Comment:
Brace should be on the same line per PSR-2
https://www.php-fig.org/psr/psr-2/#51-if-elseif-else